py/selenium/webdriver/remote/webdriver.py

```diff
@@ -225,7 +225,7 @@ def __exit__(
     @contextmanager
     def file_detector_context(self, file_detector_class, *args, **kwargs):
         """Overrides the current file detector (if necessary) in limited
-        context. Ensures the original file detector is set afterwards.
+        context. Ensures the original file detector is set afterward.
```

Comment:
This is a difference of English variants.  Afterward is US English, afterwards is GB English.  I don't think this is "wrong"